.k8s/base/redis/pvc.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: redis
+spec:
+  accessModes:
+  - ReadWriteOnce
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)